public/app/features/dashboard/dashgrid/PanelHeader/PanelHeaderMenuTrigger.tsx

```diff
@@ -40,13 +40,14 @@ export const PanelHeaderMenuTrigger: FC<Props> = ({ children, ...divProps }) =>
   );
 };
 
-function isClick(current: CartesianCoords2D, clicked: CartesianCoords2D): boolean {
-  return clicked.x === current.x && clicked.y === current.y;
+function isClick(current: CartesianCoords2D, clicked: CartesianCoords2D, deadZone = 3.5): boolean {
+  const clickDistance = Math.sqrt((current.x - clicked.x) ** 2 + (current.y - clicked.y) ** 2);
```

Comment:
I think here we can add as a comment the description you mention in the PR, it will help in the future to read this code 😄 

> A "deadzone" radius of 3.5 pixels is added so that if the cursor is moved within this radius between mousedown and mouseup, it's still considered a click and not a drag